mlflow/utils/__init__.py

```diff
@@ -172,3 +173,11 @@ def _inspect_original_var_name(var, fallback_name):
 
     except Exception:
         return fallback_name
+
+
+def load_class(kls):
+    parts = kls.split(".")
+    module = ".".join(parts[:-1])
+    m = importlib.import_module(module)
+    name = parts[-1]
+    return getattr(m, name)
```

Comment:
nit: this can be simplified using `rsplit`:

https://github.com/mlflow/mlflow/blob/d79a81671f50bf2b4d3105160912fbbee742cdec/mlflow/xgboost/__init__.py#L278-L279